feat: bump cloud-toolkit and improve doc (#15)
ForestAdmin/cloud-customizer: README.md

and install the dependencies

`yarn install` or `npm install`

then build the project

`yarn build` or `npm run build`

> **You can found in your `package.json` some scripts to help you to develop your customizations.

Reviewer: ```suggestion
**You can found in your `package.json` some scripts to help you to develop customizations.
```

> You can replace them with your own scripts if you want by using directly the `forest-cloud` command.**

Reviewer: ```suggestion
You can replace them with your own scripts, by using directly the `forest-cloud` command.**
```

# Write your customizations

> **[Read the developer guide](https://docs.forestadmin.com/developer-guide-agents-nodejs/agent-customization/agent-customization)** to learn how to write your customizations.

Reviewer: ```suggestion
**[Read the developer guide](https://docs.forestadmin.com/developer-guide-agents-nodejs/agent-customization/agent-customization)** to learn how to write customizations.
```

# Publish your customizations

To publish your customizations, you need to build, package and publish your customizations.
The build command will compile your customizations, the package command will create a package and the publish command will publish your package to your project.

```bash

> forest-cloud build

Reviewer: Should we indicate the version with `npx @forestadmin/cloud-toolkit`  here? Chances are that the package won't be globally installed.

Author: In the previous section I ask to the user to install the dependencies, it means we can use the version locally by adding npx in front of each commands

forest-cloud package
forest-cloud publish
```

You can also use `yarn` or `npm` and run `forestadmin:build:package:publish` script.

# Update typings

`forest-cloud update-typings`

You can also use `yarn` or `npm` and run `forestadmin:update-typings` script.

This command will update typings according to the structure of your database and your current code customizations.

> The typings are here to provide autocompletion in your IDE to help you write faster and less error in your code.

Reviewer: ```suggestion
The typings are here to provide autocompletion in your IDE to help you write faster and avoid errors in your code.
```
